constants.js

```diff
@@ -48,10 +48,12 @@ export const titles = [
   {label: 'Junior Software Engineer', minPoints: 0, maxPoints: 24},
   {label: 'Software Engineer', minPoints: 25, maxPoints: 53},
   {label: 'Senior Software Engineer', minPoints: 54, maxPoints: 87},
-  {label: 'Tech Leader', minPoints: 88, maxPoints: 115},
-  {label: 'Engineering Manager', minPoints: 88, maxPoints: 115},
-  {label: 'Software Architect', minPoints: 116},
-  {label: 'Head of Engineering', minPoints: 116}
+  {label: 'Tech Leader', minPoints: 88, maxPoints: 159},
+  {label: 'Engineering Manager', minPoints: 88, maxPoints: 159},
+  {label: 'Product Owner', minPoints: 88, maxPoints: 159},
```

Comment:
@AlaaSarhan the product team is planning to create a separate version of this for themselves and we will integrate this into one system... 

Adding product here is tricky for me, as I really feel this would send a wrong message for devs, and we should go more in the direction of Engineering Specialists - Like this Product Engineering Lead role I linked (See version one in the history, not the latest) and to create like Specialist lane a bit... (from there start growing the Guild concept).

If we add Product stuff here the matrices used would be muddled by the 2 roles, for example: Why would backend skills contribute to Product skills and so on...

Reply:
In my mind, a backend dev skill is necessary to some degree if the Product Manager is managing a software product, going with the reasoning in my previous reply.

In a sense, it is about the Product Manager having domain expertise (I think they always should).

If we want to go in the direction of specialization, then yes this change does not make sense, and Medium original approach fits well.

A Product Developer, or Prinicipal Product Developer role might work as bridge level role, between engineering and product teams.

I was about to use Product Devoper actually, but I feared that introducing a completely new role that isn't so clear yet, and feared as much that it might create more noise with yet-another role that need to be tested and justified. I kind have the same fear with the positions from Meetup.

Now I fear that maybe we are over-engineering the whole thing haha.

Final note for fairness, I have doubts that specialization is what we need at the moment. And that's only because I think that our projects are not sustainable on their own yet (no clear business model), which means they require more experimentation and testing (a.k.a startup phase).